### ISSUE TITLE
Notifications should be read by screen reader

### ISSUE THREAD

**Issue by PVince81** [nextcloud/nextcloud-vue#1582]
I think toast notifications should be read by screen readers when they appears.
Maybe there's a way to integrate this into the component.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alert_role

The "polite" attribute could be set depending whether it's an error or warnings.

In some cases like in Talk, not sure if messages like "connection to signaling server is taking longer" is worth reading out. Maybe need an extra flag to prevent reading those ?
